go.mod

```diff
@@ -13,10 +13,14 @@ require (
 	github.com/docker/docker v27.5.1+incompatible
 	github.com/docker/go-units v0.5.0
 	github.com/dolmen-go/contextio v0.0.0-20200217195037-68fc5150bcd5
-	github.com/elastic/beats/v7 v7.0.0-alpha2.0.20250218044933-ef79280d53a2
+	github.com/elastic/beats/v7 v7.0.0-alpha2.0.20250310095219-198197d022ad
```

Comment:
This commit needs to point to the head of the Beats 9.0 branch. It cannot point to main, that will bring in changes not targeted to 9.0.